OracleOpenJDK/18/Dockerfile

```diff
@@ -21,11 +21,11 @@
 #		$ bash build.sh
 #
 
-FROM oraclelinux:8
+FROM oraclelinux:8 
 
 MAINTAINER Aurelio Garcia-Ribeyro <aurelio.garciaribeyro@oracle.com>
 
-ENV JAVA_PKG=https://download.java.net/java/GA/jdk18.0.1/3f48cabb83014f9fab465e280ccf630b/10/GPL/openjdk-18.0.1_linux-x64_bin.tar.gz \
+ENV JAVA_URL=https://download.java.net/java/GA/jdk18.0.1.1/65ae32619e2f40f3a9af3af1851d6e19/2/GPL \
```

Comment:
Just curious... Does download.java.net not also have a "latest" URL as download.oracle.com does?

Reply:
download.java.net does not have the "latest" option as download.oracle.com does.  It is something that we have considered and might still add in the future.   If that happens, I will update the scripts accordingly